tcms/testplans/views.py

```diff
@@ -711,15 +711,16 @@ def clone(request, template_name='plan/clone.html'):
     '''Clone testplan'''
     SUB_MODULE_NAME = 'plans'
 
-    if not request.REQUEST.get('plan'):
+    req_data = request.GET or request.POST
+    if 'plan' not in req_data:
```

Comment:
There is no test which covers the jump from line 715 to 723, iow no test in which 'plan' is a parameter sent via the request! Current available tests will exit this method via the `return` on line 716.